.github/ISSUE_TEMPLATE/type_bug.yml

```diff
@@ -58,9 +58,9 @@ body:
         - Shell
         - Editor
         - Test Framework
-        - Obsevability
+        - Observability
         - CICD
-        - Other
+        - Other area
```

Comment:
I think having `Other` here is better. These labels should be changed as well to these `Runtime`, `API Docs`, `gRPC Tool`, and `SemVer Validator Tool` in all the relevant places.